### ISSUE TITLE
Remove big binaries and help files from git

### ISSUE THREAD

**Issue by tbolon** [Wagnerp/Krypton-Toolkit-Suite-NET-Core#24]
It is not recommended to store large binaries into git.

Because git is architectured to store every version of every file, everytime you push a new large binary, you simply increase the size of the repository for the same amount.

Currently, this repository is already larger than 200MB compressed.

It is recommended to always push source files, and let every developer being able to recreate binaries from his disk, or during a specific build process.

My suggestion here is to remove (and add to .gitignore) all help related files.

- [ ] remove and ignore the .chm file (66MB)
- [ ] remove the output/ directory and add to .gitignore

I need your confirmation and advice about what could be removed and added to .gitignore: that the chm can be generated from the sandcastle project in visual studio, and that the Help/Output/ directory can be generated from docfx ?

After that, I will try to find a way to also remove these binaries from the git history, so the project size could be reduced.

**Comment by tbolon** [comment]
Another topic of discussion: is the .chm help file still relevant? Now that the API documentation seems to be generated by docfx, is the Sandcastle documentation still useful ? Based on my experience, it seems the chm format is really legacy and not used anymore.

**Comment by Wagnerp** [comment]
The `chm` has been replaced with the online help, not sure if it's routed though?

**Comment by Wagnerp** [comment]
Hi @tbolon 

I'm going to have to reinstate the `docfx\Output` folder, as by removing these files, it has broken the online help.

**Comment by tbolon** [comment]
Yes I have seen that. You are right, the help\output folder is used and should have been preserved.

**Comment by Wagnerp** [comment]
@Smurf-IV 

Where do I update the year from 2019 to 2020 (making the help more relevant)?

**Comment by Smurf-IV** [comment]
@Wagnerp towards the bottom of the following file
https://github.com/Wagnerp/Krypton-Toolkit-Suite-NET-Core/blob/master/Help/DocFX/docfx.json
